borg/helpers.py

```diff
@@ -669,7 +671,10 @@ def decorated_function(*args):
 @memoize
 def uid2user(uid, default=None):
     try:
-        return pwd.getpwuid(uid).pw_name
+        if sys.platform != 'win32':
+            return pwd.getpwuid(uid).pw_name
+        else:
+            return os.getlogin()
```

Comment:
nobody/nobody and permission bits = 0? The actual permissions for Windows would be in the ACLs, and we really shouldn't even try to map those to unix permissions let alone users.